src/lib/stitching/gravity/stitching.ts

```diff
@@ -176,6 +178,25 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        descriptionFormatted: {
+          fragment: gql`
+            ... on ArtistSeries {
+              description
+            }
+          `,
+          resolve: async ({ description }, { format }) => {
+            if (!isExisty(description) || typeof description !== "string")
+              return null
+
+            const formats = {
```

Comment:
I couldn't figure out a way around this even though this [repeats the definition](https://github.com/artsy/metaphysics/blob/master/src/schema/v2/input_fields/format.ts) of a `GraphQLEnumType` defined elsewhere.

Reply:
I don't think it matters too much (these arguments are unlikely to change?), but theoretically on could import that type and use it directly. Something like:

```
const { type: formatType } = Format
const desiredFormat = type.getValues().find(e => e.name === format).value

return formatMarkdownValue(description, desiredFormat)
```